packages/next/server/config-shared.ts

```diff
@@ -382,6 +382,13 @@ export interface NextConfig extends Record<string, any> {
           exclude?: string[]
         }
     styledComponents?: boolean
+    emotion?:
+      | boolean
+      | {
+          sourceMap?: boolean
```

Comment:
Does emotion uses its own sourcemap?
SWC is designed to support emitting sourcemap for any files just by using `Span`.

`swc_ecma_codegen` is enough to generate source maps for css files.